pkg/api/object.go

```diff
@@ -44,7 +73,7 @@ func (h *Handler) CreateObjectStore(w http.ResponseWriter, r *http.Request) {
 // DELETE
 // /objectstore
 func (h *Handler) RemoveObjectStore(w http.ResponseWriter, r *http.Request) {
-	if err := h.config.ClusterHandler.RemoveObjectStore(); err != nil {
+	if err := h.config.ClusterHandler.RemoveObjectStore(defaultObjectStoreName); err != nil {
```

Comment:
what if the object store was created with an user-specified name? This will fail to delete.

Reply:
it wasn't plumbed yet for the rest api to support multiple. I'm adding that in the next PR